code/modules/clothing/suits/ego_gear/zayin.dm

```diff
@@ -57,3 +57,9 @@
 	desc = "An old brown jacket. What seems to be an L corp logo is stitched into the side. This logo, strangely, is not one for the company you work at"
 	icon_state = "nostalgia"
 	armor = list(RED_DAMAGE = 0, WHITE_DAMAGE = 10, BLACK_DAMAGE = 0, PALE_DAMAGE = 0)
+
+/obj/item/clothing/suit/armor/ego_gear/mailman
+	name = "Postal Service Suit"
+	desc = "A standard postman attire with a L corp badge."
```

Comment:
What? There are several issues with this... Look at the naming scheme and general descriptions of other E.G.O. suits.